test/common/pixel-tests

```diff
@@ -1,25 +1,35 @@
 #!/bin/bash
 
-set -eu
-
-SUBDIR=test/reference
-REPO=pixel-test-reference
-
-GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
-GITHUB_REPOSITORY="${GITHUB_BASE%/*}/${REPO}"
-CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
-PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
+GITHUB_REPO='pixel-test-reference'
+SUBDIR='test/reference'
 
-message() {
-    [ "${V-}" != 0 ] || printf "  %-8s %s\n" "$1" "$2"
-}
+set -eu
```

Comment:
Tiny style nitpick: This ought to stay the first line in this (and every) script. Presumably it just got moved by accident?